tests/namespaced.rs

```diff
@@ -284,10 +282,8 @@ mod namespaced {
         let local = manager.deploy_ztunnel_inpod(DEFAULT_NODE, uds_default_node.clone())?;
 
         info!("sending workload to ztunnel");
-        remote_node_server.send(server_fd).await.unwrap();
-        remote_node_server_ack.recv().await.unwrap();
-        default_node_server.send(client_fd).await.unwrap();
-        default_node_server_ack.recv().await.unwrap();
+        remote_node_server.register_workload(server_fd).await;
+        default_node_server.register_workload(client_fd).await;
```

Comment:
wip: refactoring the test harness to allow re-using the inpod stuff without as much copypaste